### PR TITLE
Switch to using local cities json

### LINKED ISSUE #68: Update GeoJSON file with correct boxes and country names
The boxes are currently coming from a GeoJSON file on S3, URL is in `templates/index.html`.

### PR DESCRIPTION
**Pull request by migurski**
* [x] Stop using S3 URL for GeoJSON.
* [x] Fix names and countries.
* [x] Generate `href` for metros.
* [x] Stop splitting on names in front-end code; use `display_name`, `id`, and `href`. Stop using `name` entirely; it’s deprecated.

Closes #68.

### CODE REVIEW

File: App/templates/index.html [hunk 1]
```diff
@@ -290,9 +286,9 @@ <h1 class="headroom text-center">metro extracts</h1>
       });
     var cities = countries.selectAll(".city").data(function(d){ return d.metros; });
     cities.enter().append("a").attr("class","city");
-    cities.text(function(d){ return d.name.split("_")[0].replace(/-/g," "); })
+    cities.text(function(d){ return d.name; })
       .attr("href",function(d){
```
Comment (migurski) [inline App/templates/index.html]: can we switch this to use the `href` property?

Comment (binx) [inline App/templates/index.html]: @migurski the href property isn't available in the cities.json file 
screen shot 2016-06-29 at 11 46 07 am

Comment (migurski) [inline App/templates/index.html]: Oh, right this is the JSON and not the GeoJSON file. Right.

…

I will cause it to be there.

Comment (migurski) [inline App/templates/index.html]: Okay looked at this a little more, and I’m not sure I understand why an href is being set at all. They’re being provided in the HTML loop on line 195; is this Javascript re-creating them from scratch? Can it instead hide/show as appropriate?

Comment (binx) [inline App/templates/index.html]: I'm using d3 to keep track of the list divs, so they aren't being shown/hidden they are being data-binded and need to be updated with the current text and href

Comment (migurski) [inline App/templates/index.html]: Okay; seems like it should be possible to re-use the provided `href` value directly? Or is this getting data from the `nestedCities` var?

Comment (binx) [inline App/templates/index.html]: it's getting data from the `nestedCities` var...

Comment (migurski) [inline App/templates/index.html]: K, got it.

**Comment by migurski** [comment]
Okay, the `href` is there, so I’ll kick this back to you to wrap up the D3 integration.

**Pull request merged**